dev-quick-start.md

```diff
@@ -139,47 +132,40 @@ This section will be updated over time as K8ssandra grows.  Take a look to enhan
 
 You may experience a `missing in charts/ directory` error message.  
 
-If so, you can 
-utilize a K8ssandra script: `./scripts/update-helm-deps.sh`. This script assists with updating dependencies for each chart in an appropriate order.  
+If so, you can utilize a K8ssandra script: `./scripts/update-helm-deps.sh`. This script assists with updating dependencies for each chart in an appropriate order.  
 
 Be sure to run this script so the `./charts` folder is properly located. 
 
-
-
   
 ### Collecting useful information 
 
 So you have an error after editing a K8ssandra configuration, or you want to inspect some things as you learn.  There are some useful commands that come in handy when needing to dig a bit deeper.  The examples assume you are using a k8ssandra namespace, but this can be adjusted as needed.
 
-Issue the following `kubectl` command to view the `Cassandra` logs.  Replace *<pod-with-cassandra>* with an actual pod instance name having the `Cassandra` container.
-
-> kubectl logs pod/<pod-with-cassandra> cassandra -n k8ssandra
+Issue the following `kubectl` command to view the `Cassandra` logs.  Replace *<cassandra-pod>* with an actual pod instance name.
 
+> kubectl logs pod/<cassandra-pod> -c cassandra -n k8ssandra
 
 
-Issue the following `kubectl` command to view `Medusa` logs.  Replace *<pod-with-medusa>* name with an actual pod instance name having the `Medusa` container.
+Issue the following `kubectl` command to view `Medusa` logs.  Replace *<cassandra-pod>* name with an actual pod instance name.
 
-> kubectl logs pod/<pod-with-medusa> -n k8ssandra
+> kubectl logs pod/<cassandra-pod> -c medusa -n k8ssandra
```

Comment:
This can be further shortened to `kubectl logs <cassandra-pod> -c medusa -n k8ssandra`